README.md

```diff
@@ -114,6 +114,14 @@ Interface | `ng g interface my-new-interface`
 Enum      | `ng g enum my-new-enum`
 Module    | `ng g module my-module`
 
+angular-cli will add reference to `components`, `directives` and `pipes` automatically in the `app.module.ts`. If you need to add this references to another custom module, follow this steps:
+
+1. `ng g module newModule` to create a new module
+2.  you should change directory with `cd newModule` into the `newModule` folder
```

Comment:
You can add another commit to your PR branch with the changes I suggested above. Once you've made those changes locally you can push your branch back to your forked repo. This will update the PR as well which will allow myself or someone else to re-review the changes. Don't worry about rebasing your changes or combining commits, we can take care of that when merging (for most cases, especially things like this within the README)